device/arista/x86_64-arista_7050_qx32/Arista-7050-Q16S64/td2-a7050-qx32-16x40G+32x10G+8x40G.config.bcm

```diff
@@ -1,5 +1,6 @@
 #/******************************************************************************
 # *
+bcm_tunnel_term_compatible_mode=1
```

Comment:
why this line of the configuration is inserted inside the comment header?

Reply:
It was added by a script. I didn't look close at the end result. I can move it if you prefer.